src/doc/stdlib.ts

```diff
@@ -48,6 +48,20 @@ export const doc_stdlib:Documentation = {
 		<tr><th>same</th><td>
 			The <code class="code">function same(first, second)</code> tests
 			whether its arguments refer to the same object.
+            <div class="example">
+            <h3>Example</h3>
+            <tscript>
+var a = 1;
```

Comment:
Better example: var a = [1,2,3];
and of course the same for d. Then the second test case will print false.